lib/gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -55,13 +76,16 @@ def ask_for_repmgrd_configuration
     def activate
       say("Configuring Replication Standby Server...")
       data_dir_empty? &&
-        generate_cluster_name &&
-        create_config_file(standby_host) &&
-        clone_standby_server &&
-        start_postgres &&
-        register_standby_server &&
-        write_pgpass_file &&
-        (run_repmgrd_configuration ? start_repmgrd : true)
+        initialize_postgresql_disk if disk
+        PostgresAdmin.prep_data_directory &&
```

Comment:
I think the check for the empty data directory should go after these because the clone will fail if it isn't empty.